<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNuevoCapitulo</class>
 <widget class="QWidget" name="dlgNuevoCapitulo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="txtNombreGeneral">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nombre general&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblLugar">
       <property name="text">
        <string>Lugar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtLugar"/>
     </item>
     <item>
      <widget class="QPushButton" name="btQuitarLugar">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Añadir lugar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tipo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboTipoCapitulo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fecha inicial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtInicial">
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Fecha final</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtFinal">
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Maestro general</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtMaestroGeneral"/>
     </item>
     <item>
      <widget class="QPushButton" name="btQuitarMaestroGeneral">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtAsistentes">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Asistentes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtNotas">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Notas&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Tomo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtTomo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Págs.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPaginas"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancel">
       <property name="text">
        <string>Can&amp;celar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/myqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtNombreGeneral</tabstop>
  <tabstop>txtLugar</tabstop>
  <tabstop>btQuitarLugar</tabstop>
  <tabstop>cboTipoCapitulo</tabstop>
  <tabstop>dtInicial</tabstop>
  <tabstop>dtFinal</tabstop>
  <tabstop>txtMaestroGeneral</tabstop>
  <tabstop>btQuitarMaestroGeneral</tabstop>
  <tabstop>txtAsistentes</tabstop>
  <tabstop>txtNotas</tabstop>
  <tabstop>txtTomo</tabstop>
  <tabstop>txtPaginas</tabstop>
  <tabstop>btCancel</tabstop>
  <tabstop>btOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
